Documentation/General/productVision/productVision.tex

```diff
@@ -1,30 +1,41 @@
-\documentclass[10pt,a4paper]{article}
-\usepackage[utf8]{inputenc}
-\usepackage{amsmath}
-\usepackage{amsfonts}
-\usepackage{amssymb}
-\usepackage{cite}
-\usepackage{hyperref}
-\usepackage{graphicx}
-\usepackage{siunitx}
-
-\begin{document}
-
-\title{Product Vision\\
-	\begin{small}
-	The why, who and what
-	\end{small}
-}
-% Question: should this be our names or organization name?
-\author{MoodCat.me}
-\maketitle
-
-\section{Why should we build MoodCat?}
+\chapter{Introduction}
+
+When people want to party, they want to listen to loud bass-heavy hype music.
+When people feel down, they want to listen to slower emotional music.
+When people feel sleepy, they want to listen to calm music.
+The type of music people want to listen to is based on their mood, there's no doubt about that.
+
+Our goal is to use this relationship between mood and music,
+combine it with a music recommendation system and make a music service
+that will influence and be influenced by the listeners over time.
+This is our main focus point.
+
+Besides this we're going to add a social component to listening mood based music.
+It doesn't matter which genre a song belongs to or which mood a listener is in;
+Listening music together is more joyful, since you can share experiences.
+We want to make a music service that connects people that are in the same mood and make them enjoy the songs together.
+Users of our music service will be able to join a room based on a chosen mood.
+This room will also have a chat to be able to interact with other people who are in the same mood.
+
+Listeners will provide feedback on the change of music by clicking buttons to (dis)like songs.
+This feedback will influence the song-selection system to further enhance the future user experience.
+
+The combination of a mood-based song-selection algorithm and a chat to be able to interact with other users makes MoodCat a successful product.
+These crucial components will satisfy the previously defined goal and will therefore be our main focus point of development.
+
+\section{Timeframe}
+This product will be developed over a span of 11 weeks, with no budget.
+Every week will consist of a sprint which will result in a working product to be shown at the demonstration on fridays.
+
+\chapter{Why should we build MoodCat?}
 Various competitors provide a system that can offer users popular songs or songs that are similar to songs the user already listened to.
 These recommendations mostly consist of songs that are popular or users with simmilar preferences listened to.
 This leaves a huge amount of music produced by a lot of artists undiscovered.
 
-Secondly, most music platforms do not offer user interaction.
+Secondly, most music platforms do not offer explicit user interaction.
```

Comment:
Explicit in the context of content may have a very different meaning (which I would love to see in the conversations, but this is a whole other story)